pkg/manifest/os.go

```diff
@@ -93,35 +93,36 @@ type OSCustomizations struct {
 	ShellInit []shell.InitFile
 
 	// TODO: drop osbuild types from the API
-	Firewall            *osbuild.FirewallStageOptions
-	Grub2Config         *osbuild.GRUB2Config
-	Sysconfig           []*osbuild.SysconfigStageOptions
-	SystemdLogind       []*osbuild.SystemdLogindStageOptions
-	CloudInit           []*osbuild.CloudInitStageOptions
-	Modprobe            []*osbuild.ModprobeStageOptions
-	DracutConf          []*osbuild.DracutConfStageOptions
-	SystemdUnit         []*osbuild.SystemdUnitStageOptions
-	Authselect          *osbuild.AuthselectStageOptions
-	SELinuxConfig       *osbuild.SELinuxConfigStageOptions
-	Tuned               *osbuild.TunedStageOptions
-	Tmpfilesd           []*osbuild.TmpfilesdStageOptions
-	PamLimitsConf       []*osbuild.PamLimitsConfStageOptions
-	Sysctld             []*osbuild.SysctldStageOptions
-	DNFConfig           []*osbuild.DNFConfigStageOptions
-	DNFAutomaticConfig  *osbuild.DNFAutomaticConfigStageOptions
-	YUMConfig           *osbuild.YumConfigStageOptions
-	YUMRepos            []*osbuild.YumReposStageOptions
-	SshdConfig          *osbuild.SshdConfigStageOptions
-	GCPGuestAgentConfig *osbuild.GcpGuestAgentConfigOptions
-	AuthConfig          *osbuild.AuthconfigStageOptions
-	PwQuality           *osbuild.PwqualityConfStageOptions
-	OpenSCAPConfig      *osbuild.OscapRemediationStageOptions
-	NTPServers          []osbuild.ChronyConfigServer
-	WAAgentConfig       *osbuild.WAAgentConfStageOptions
-	UdevRules           *osbuild.UdevRulesStageOptions
-	WSLConfig           *osbuild.WSLConfStageOptions
-	LeapSecTZ           *string
-	FactAPIType         *facts.APIType
+	Firewall                 *osbuild.FirewallStageOptions
+	Grub2Config              *osbuild.GRUB2Config
+	Sysconfig                []*osbuild.SysconfigStageOptions
+	SystemdLogind            []*osbuild.SystemdLogindStageOptions
+	CloudInit                []*osbuild.CloudInitStageOptions
+	Modprobe                 []*osbuild.ModprobeStageOptions
+	DracutConf               []*osbuild.DracutConfStageOptions
+	SystemdUnit              []*osbuild.SystemdUnitStageOptions
+	Authselect               *osbuild.AuthselectStageOptions
+	SELinuxConfig            *osbuild.SELinuxConfigStageOptions
+	Tuned                    *osbuild.TunedStageOptions
+	Tmpfilesd                []*osbuild.TmpfilesdStageOptions
+	PamLimitsConf            []*osbuild.PamLimitsConfStageOptions
+	Sysctld                  []*osbuild.SysctldStageOptions
+	DNFConfig                []*osbuild.DNFConfigStageOptions
+	DNFAutomaticConfig       *osbuild.DNFAutomaticConfigStageOptions
+	YUMConfig                *osbuild.YumConfigStageOptions
+	YUMRepos                 []*osbuild.YumReposStageOptions
+	SshdConfig               *osbuild.SshdConfigStageOptions
+	GCPGuestAgentConfig      *osbuild.GcpGuestAgentConfigOptions
+	AuthConfig               *osbuild.AuthconfigStageOptions
+	PwQuality                *osbuild.PwqualityConfStageOptions
+	OpenSCAPAutotailorConfig *osbuild.OscapAutotailorStageOptions
```

Comment:
This is not needed, since the tailoring information is already part of `OpenSCAPConfig`

Reply:
Right, my bad. I didn't check everything thoroughly and switched the osbuild stage options for the BP customizations structure.... :facepalm: